src/Perimeterx.php

```diff
@@ -59,9 +59,7 @@ public static function Instance(array $pxConfig = [])
 
     private function __construct(array $pxConfig = [])
     {
-        if (!function_exists("mcrypt_encrypt")) {
-            throw new PerimeterxException(PerimeterxException::$MCRYPT_MISSING);
-        }
+        
```

Comment:
why did we remove the test if mcrypt exists?

Reply:
mcrypt is no longer in use. there is a different Jira to remove all instances that reference it.